docker/mainnet/litecoin/litecoin.mainnet.conf

```diff
@@ -0,0 +1,16 @@
+port=12333
+
+rpcbind=0.0.0.0
+rpcport=12332
+
+rpcallowip=172.100.2.100
+
+addresstype=legacy
+
+txindex=1
```

Comment:
Why do we need txindex in litecoin?

Reply:
Not sure, but it was in testnet and simnet in previous installations